tools/test/scripts/nb_test/subscribe.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I'd remove this

Reply:
@Andrea-Campanella Did you mean the shebang or the whole script?
I agree with removing the whole script - not much to script and besides it can't handle arguments for rollback etc

Reply:
the whole thing, did not find a "file" comment